core/src/main/java/com/datastrato/gravitino/connector/BaseCatalog.java

```diff
@@ -57,6 +65,13 @@ public abstract class BaseCatalog<T extends BaseCatalog>
   // The object you used is not stable, don't use it unless you know what you are doing.
   @VisibleForTesting public static final String CATALOG_OPERATION_IMPL = "ops-impl";
 
+  // Different catalogs may have different authorization provider, this variable is used as a
+  // key in properties of catalogs to inject custom authorization to Gravitino.
+  public static final String AUTHORIZATION_PROVIDER = "authorization-provider";
```

Comment:
As I mentioned in the previous comment, it would be better to  move the definition to `Catalog.java` in API module, where all the catalog related common properties are defined.

Also update the doc to mention about this property.

Reply:
OK, I move `AUTHORIZATION_PROVIDER` into `Catalog.java`.